build/3-apillon-api.md

```diff
@@ -1158,3 +1158,714 @@ curl --location --request GET "https://api.apillon.io/hosting/websites/:websiteU
+|-------|--------------------------------------------------------------------| -------- |
+| uuid  | Unique key of collection. Key is displayed in developer dashboard. | true     |
+
+#### Body fields
+
+| Name    | Type     | Description                    | Required |
+|---------|----------|--------------------------------| -------- |
+| address | `string` | Wallet address of a new owner. | true     |
+
+#### Possible errors
+
+| Code     | Description                                                                       |
+| -------- |-----------------------------------------------------------------------------------|
+| 40012003 | Contract can't be transferred to wallet address that already owns this collection. |
+| 40012004 | Transfer transaction already exists.                                              |
+| 40300000 | Not allowed to access collection                                                  |
+| 50012002 | Collection doesn't exist, wasn't deployed or was already transferred.              | 
+| 50012004 | Collection transfer failed.                                                       | 
+
+#### Response
+
+Response payload is described [under Response Fields above](#get-nft-collection).
+
+  </div>
+  <div class="split_side">
+
+  <CodeGroup>
+  <CodeGroupItem title="cURL" active>
+
+```sh
+curl --location 'https://api.apillon.io/nfts/collections/:uuid/transfer' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Basic :credentials' \
+--data '{"address": "0x452101C96A1Cf2cBDfa5BB5353e4a7F235241551"}'
+```
+
+  </CodeGroupItem>
+  </CodeGroup>
+  <CodeGroup>
+  <CodeGroupItem title="Response">
+
+```json
+{
+    "id": "b5935c73-204d-4365-9f9a-6a1792adab5b",
+    "status": 200,
+    "data": {
+        "id": 23,
+        "status": 2,
+        "collectionUuid": "d6355fd3-640d-4803-a4d9-79d875abcb5a",
+        "projectUuid": "b8d3cb86-fde5-4d96-81af-bf47f8703e22",
+        "symbol": "NFT",
+        "name": "NFT Collection",
+        "description": null,
+        "maxSupply": 1000,
+        "mintPrice": 0.1,
+        "bucketUuid": "a9425ff7-4802-4a38-b771-84a790112c30",
+        "baseUri": "https://ipfs.apillon.io/metadata/",
+        "baseExtension": ".json",
+        "isDrop": true,
+        "isSoulbound": false,
+        "isRevokable": true,
+        "dropStart": 1687251003,
+        "reserve": 5,
+        "royaltiesFees": 0,
+        "royaltiesAddress": "0x4156edbafc5091507de2dd2a53ded551a346f83b",
+        "collectionStatus": 0,
+        "contractAddress": "0x452101C96A1Cf2cBDfa5BB5353e4a7F235241557",
+        "transactionHash": "0x6b97424de3367cd0335b08265787b83053b62bee2d1c8bec1f776936bea4fb26",
+        "deployerAddress": "0x4156edbafc5091507de2dd2a53ded551a346f83b",
+        "chain": 1287,
+        "minted": 0
+    }
+}
+```
+
+  </CodeGroupItem>
+  </CodeGroup>
+  </div>
+</div>
+
+
+### Mint Collection NFTs
+
+> Mint specified amount of NFTs to a wallet address provided in request.
```

Comment:
Would add to description that if the collection is set as `isDrop` this function can only mint the `reserved` nfts.